onnx/defs/tensor/defs.cc

```diff
@@ -628,7 +628,13 @@ ONNX_OPERATOR_SET_SCHEMA(
             "where r = rank(input).",
             AttributeProto::INT,
             static_cast<int64_t>(0))
-        .SetDoc(Split_ver13_doc)
+        .Attr(
+            "num_outputs",
+            "Number of outputs to split equal parts of the tensor into. "
+            "Can be overridden by input 'split'.",
+            AttributeProto::INT,
+            static_cast<int64_t>(1))
```

Comment:
I think the attribute should be optional.

Reply:
Changed it to optional.